.github/workflows/autodeploy.yaml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Storybook to S3
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
```

If you want to test this you can remove the branch restriction and then the action will run on this PR every time you push a change